ProductsWebApi/ProductsWebApi/Controllers/AuthController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+
+namespace ProductsWebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IUserService _userService;
+
+        public AuthController(IConfiguration configuration, IUserService userService)
+        {
+            _configuration = configuration;
+            _userService = userService;
+        }
+
+        [HttpPost]
+        [ActionName("RequestToken")]
+        public IActionResult RequestToken([FromBody]Users userLoginModel)
+        {
+            try
+            {
+                if (_userService.IsValidUser(userLoginModel))
+                {
+                    return Ok(GenerateJwtToken());
+                }
+                else
```

Comment:
Again invert if, please. You should avoid, if along with else as much as possible. Especially if you are returning inside if.

Reply:
Changed to invert if